colorclade/_biopython_draw_colorclade_tree.py

```diff
@@ -75,6 +79,13 @@ def biopython_draw_colorclade_tree(
     mpl_Axes
         The matplotlib Axes object with the tree drawn.
     """
+    if isinstance(label_tips, bool):
+        label_tips_val = label_tips
```

Comment:
This is a really nice way to handle the bool special cases. We can deal with the late binding nicely using default param to force capture (instead of another variable) https://stackoverflow.com/a/2295372